src/adapters/mc/complianceengine/Baseline.c

```diff
@@ -44,7 +52,7 @@ int BaselineMmiGet(const char* componentName, const char* objectName, char** pay
         return EINVAL;
     }
 
-    int result = ComplianceEngineMmiGet(gComplianceEngine, componentName, objectName, payload, payloadSizeBytes);
+    int result = ComplianceEngineMmiGet(GetHandle(), componentName, objectName, payload, payloadSizeBytes);
```

Comment:
The lifecycle of the handle to our engine is managed by the caller, we shouldn't be initialising it in Get/Set if it hasn't been initialised or if it's been freed.

Can you outline what issue you faced? (this could have been part of the commit message and PR description)